modules/creating-manifest-file-multiple-interfaces-br-ex.adoc

```diff
@@ -0,0 +1,177 @@
+
+* Drains nodes in singular order based on the selected machine config pool.
+* Injects Ignition configuration files into each node, so that each node receives the additional `br-ex` bridge network configuration.
+* Restarts the node with the modified override file.
+* Executes the `configure-ovs.sh` shell script that references the new interface definition.
+* Adds `br-ex` and `br-ex1` to the host node.
+* Uncordons the nodes. After all the nodes return to the `Ready` state, the Operator applies the `k8s.ovn.org/l3-gateway-config` annotation to each node.
+
+For more information about useful situations for the additional `br-ex` bridge and a situation that always requires the default `br-ex` bridge, see "Configuration for a localnet topology".
+
+.Procedure
+
+. Convert the rule to Base64 encoded by running the following command:
++
+[source,terminal]
+----
+$ base64 /path/to/file/
+----
+
+. Complete one of the following steps to define a secondary interface for your network. Ensure that the secondary interface is not bound to the `br-ex` bridge.
++
+[IMPORTANT]
+====
+Do not use the Kubernetes NMState Operator to define or a `NodeNetworkConfigurationPolicy` (NNCP) manifest file to define the secondary interface.
+====
++
+.. Define a secondary interface in a machine config manifest file:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 11-master-bond0
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;base64,<encoded_base64_string> <1>
+        path: /etc/NetworkManager/system-connections/bond0.nmconnection
+        filesystem: root
+        mode: 0600
+      - contents:
+# ...
+----
+<1> The Base64 encoded string that you have generated in the previous step.
++
+.. Create a connection profile for the bonded secondary interface and save the file in the `/etc/NetworkManager/system-connections/secondary-iface.nmconnection` path. For example, create the `bond1.nmconnection` file in your local directory with the following content:
++
+[source,ini]
+----
+[connection]
+id=eno2
+type=ethernet
+interface-name=eno2
+master=bond0
+slave-type=bond
+autoconnect=true
+autoconnect-priority=20
+
+[ethernet]
+mtu=9000
+----
+
+. Create an Open vSwitch (OVS) bridge, `br-ex1`, on nodes by using OVN-Kubernetes network plugin to create an `extra_bridge` file`. Ensure that you save the file in the `/etc/ovnk/extra_bridge` path of the host. The file must state the interface name that supports the additional bridge and not the default interface that supports `br-ex`, which holds the primary IP address of the node. 
++
+.Example configuration for the `extra_bridge` file, `/etc/ovnk/extra_bridge`, that references a secondary interface that is attached to a VLAN
+[source,text]
+----
+ens4f1.1001
```

Comment:
since we're giving an example of `bond1` as our interface above, let's modify this to point to `bond1` for consistency

Reply:
:heavy_check_mark: